Dockerfile

```diff
@@ -2,10 +2,15 @@ FROM thmhoag/steamcmd:latest
 
 USER root
 
-RUN apt-get update && \
-    apt-get install -y curl cron bzip2 perl-modules lsof libc6-i386 lib32gcc1 sudo
+RUN apt-get update \
+    && apt-get install -y curl cron bzip2 perl-modules lsof libc6-i386 lib32gcc1 sudo \
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/* \
+    && rm -rf /var/tmp/*
 
-RUN curl -sL "https://raw.githubusercontent.com/FezVrasta/ark-server-tools/v1.6.54/netinstall.sh" | bash -s steam && \
+RUN curl -sL https://git.io/arkmanager | bash -s steam && \
```

Comment:
Can we make this configurable via a build variable? The main reason this is hard coded is because I have had issues in the past where changes to `ark-server-tools` were actually breaking changes for this image. 

Another idea might be to add an environment variable that, if set, updates `ark-server-tools` on startup (with the caveat that breaking changes are possible).

Reply:
Sure, I understand and goal should be to provide always a working image.

Suggestion: let's work together to make it configurable and have several docker images, with version number and "latest". Then everybody can choose either a versioned image or "latest".

Example for Dockerfile arguments:
```
ARG ARKMANAGER_VERSION=latest
[...]
RUN if [ $ARKMANAGER_VERSION = "latest" ]; then curl -sL https://git.io/arkmanager; else curl -sL ...; fi | bash -s steam && \
[...]
```

Same goes for the base steamcmd image. We can build for multiple Ubuntu versions, like I did in some of my projects, like my (soon to be obsolete) ARK image: https://github.com/DrPsychick/docker-linuxgsm-ark/blob/928590d95e9be79e4175d7b903611244620e72a5/.travis.yml#L10

Reply:
So, I got it to work... BUT it seems the arkmanager installer always uses the latest version, so the `AMG_VERSION` variable won't have any effect. Or am I missing something?

```
Step 11/29 : RUN curl -sL https://raw.githubusercontent.com/arkmanager/ark-server-tools/$AMG_VERSION/netinstall.sh | bash -s steam
 ---> Running in 7ff30764ae9d
Getting latest release...
Latest release is v1.6.58
```